decorator.rb

```diff
@@ -0,0 +1,10 @@
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
```

Comment:
- [ ] your decorator class is missing a`ttr_accessor :nameable` kindly add it to match the requirements 🙏